config/database.yml

```diff
@@ -5,6 +5,8 @@ development:
   adapter: postgresql
   database: theodinproject
   host: localhost
+  username:
```

Comment:
Can we remove this file from the PR please, `git checkout origin/master config/database.yml` should do it